src/events/event-bus.ts

```diff
@@ -1,13 +1,15 @@
+// @ts-nocheck
```

Comment:
why?

Reply:
The file is not properly typed yet. We can do this in a followup (it's a bit tricky though because of all the window manipulation in there)